etl/scripts-ccao-data-warehouse-us-east-1/spatial/spatial-environment-ohare_noise.R

```diff
@@ -60,8 +60,10 @@ names(noise_levels) <- columns
 
 # Clean NAs and remove inactive sites
 noise_levels <- noise_levels %>%
-  na_if("n/a") %>%
-  na_if("--") %>%
+  mutate(
+    across(where(is.character), ~ na_if(.x, "n/a")),
+    across(where(is.character), ~ na_if(.x, "--"))
+    ) %>%
```

Comment:
`na_if` throws errors these days if you pass it a dataframe rather than a vector.